src/app/(auth)/email-verified/page.tsx

```diff
@@ -4,16 +4,19 @@ import { useRouter } from 'next/navigation';
 import { H2 } from '@/styles/text';
 import { SpacerDiv } from '@/app/(auth)/styles';
 import BigButton from '@/components/BigButton';
+import NavBar from '@/components/NavBar';
 
 export default function EmailVerified() {
   const { push } = useRouter();
-
   return (
-    <SpacerDiv>
-      <H2>Your email has been verified!</H2>
-      <BigButton type="button" onClick={() => push('/onboarding/roles')}>
-        Go to Onboarding
-      </BigButton>
-    </SpacerDiv>
+    <>
+      <NavBar />
```

Comment:
since the NavBar component was already added in the auth layout, we don't need to add it again on this page! it currently looks like this: 
![Uploading Screen Shot 2024-03-06 at 9.43.38 PM.png…]()

Reply:
Fixed